tests/encryption_/models.py

```diff
@@ -0,0 +1,88 @@
+from django.db import models
+
+from django_mongodb_backend.fields import (
+    EncryptedBigIntegerField,
+    EncryptedBinaryField,
+    EncryptedBooleanField,
+    EncryptedCharField,
+    EncryptedDateField,
+    EncryptedDateTimeField,
+    EncryptedDecimalField,
+    EncryptedEmailField,
+    EncryptedFloatField,
+    EncryptedGenericIPAddressField,
+    EncryptedIntegerField,
+    EncryptedPositiveBigIntegerField,
+    EncryptedPositiveSmallIntegerField,
+    EncryptedSmallIntegerField,
+    EncryptedTextField,
+    EncryptedTimeField,
+    EncryptedURLField,
+)
+
+EQUALITY_QUERY = {"queryType": "equality"}
+RANGE_QUERY = {"queryType": "range"}
+
+
+class Appointment(models.Model):
+    time = EncryptedTimeField(queries=EQUALITY_QUERY)
+
+    class Meta:
+        required_db_features = {"supports_queryable_encryption"}
+
+
+class Billing(models.Model):
+    cc_type = EncryptedCharField(max_length=20, queries=EQUALITY_QUERY)
+    cc_number = EncryptedBigIntegerField(queries=EQUALITY_QUERY)
+    account_balance = EncryptedDecimalField(max_digits=10, decimal_places=2, queries=RANGE_QUERY)
+
+    class Meta:
+        required_db_features = {"supports_queryable_encryption"}
+
+
+class PatientPortalUser(models.Model):
+    ip_address = EncryptedGenericIPAddressField(queries=EQUALITY_QUERY)
+    url = EncryptedURLField(queries=EQUALITY_QUERY)
+
+    class Meta:
+        required_db_features = {"supports_queryable_encryption"}
+
+
+class PatientRecord(models.Model):
+    ssn = EncryptedCharField(max_length=11, queries=EQUALITY_QUERY)
+    birth_date = EncryptedDateField(queries=RANGE_QUERY)
+    profile_picture = EncryptedBinaryField(queries=EQUALITY_QUERY)
+    patient_age = EncryptedIntegerField("patient_age", queries=RANGE_QUERY)
+    weight = EncryptedFloatField(queries=RANGE_QUERY)
```

Comment:
Can we also have a test with `RANGE_QUERY` holding `min, max`?

Reply:
done